lib/fluent/config/dsl.rb

```diff
@@ -96,6 +101,18 @@ def method_missing(name, *args, &block)
           self
         end
 
+        def include(*args)
+          ::Kernel.raise ::ArgumentError, "#{name} block requires arguments for include path" if args.nil? || args.size != 1
+          if args.first =~ /\.rb$/
+            path = File.expand_path(args.first)
+            data = File.read(path)
+            self.instance_eval(data, @proxy.include_basepath)
```

Comment:
The 2nd argument of `instance_eval` should be the filename (`path`), not base path.